src/main/resources/application.conf

```diff
@@ -1,5 +1,16 @@
+serverAddress {
+  interface = "127.0.0.1"
+  port = 9076
+}
+
+discovery {
+  bootstrapNodes = [
+    "enode://08aa4a60712b17b4144cc6e97a6fbb254007d2e73e8eb83c0b2757cb696b3b8464a299ca504e9907769ac7ca398993282524c4c0afa3cd07d15250c179a875e0@127.0.0.1:60303?discport=0"
```

Comment:
@ntallar I assume that this is your local node with specific public key, should this vale be in main repo?